vars/main.yml

```diff
@@ -66,6 +66,9 @@ __mssql_ha_private_key_dest: /var/opt/mssql/data/{{ mssql_ha_cert_name }}.pvk
 __mssql_ad_kinit_user: >-
   {{ mssql_ad_kerberos_user }}@{{ ad_integration_realm | upper }}
 
+# OS has tuned with the extra "mssql" profile
+__mssql_tuned_supported: false
```

Comment:
This will only matter if the role ever supports anything other than the Red Hat family (RHEL, Fedora, CentOS, Rocky, Alma, etc.) and SUSE e.g. if we ever add support for Ubuntu, we won't need to add a `vars/Ubuntu.yml` in order to define `__mssql_tuned_supported` but we'll probably need to add it anyway for other variables.

Reply:
Ah, Fedora uses vars/RedHat.yml? That works through `ansible_os_family`? OK.

Reply:
> Ah, Fedora uses vars/RedHat.yml? That works through `ansible_os_family`? OK.

Yes  https://github.com/ansible/ansible/blob/40c919d7bdcadf5e17704995b5950fd78f84da7e/lib/ansible/module_utils/facts/system/distribution.py#L512